main.py

```diff
@@ -20,13 +13,12 @@
 namesperson = []  # For collection of all usernames it acts as db for user names
 while True:
     for searchterm in searchterms:
-        
         for tweet in tweepy.Cursor(api.search, q=searchterm).items(10):
             if tweet in tweets:
                 pass
             else:
-                tweets.append(tweet.text)  # AAdding the tweet to db
-                if tweet.author.id not in names:  # Checking if the same person tweeted more than once
+                tweets.append(tweet.text)  # Adding tweet to db
+                if tweet.author.id not in names:  # Checking if same user tweeted more than once
```

Comment:
let's follow pep8 and limit the line length to 79 characters.

Also, I'm sure pylint score for this script would be less than 5, lets make 9+.

Reply:
If you refer to #7, I tried to implement flake8. But couldn't 
Will try again.

Reply:
Sure.